api_directive.go

```diff
@@ -324,6 +324,20 @@ func (api *API) pushJobsTableKeys(ctx context.Context, jobs chan<- directiveJobT
 	// Get the diff between from/to directive.partitions.
 	partComp := newPartitionsComparer(fromD.TranslatePartitionsMap(), toPartitionsMap)
 
+	// Remove any partitions which are no longer assigned to this worker.
+	// TODO(tlt): currently, this is just removing the file lock on the
+	// resource; it's not actually removing the resource from the local
+	// computer. We should do that.
```

Comment:
we should... I guess it's maybe a bit unclear how to do that... we probably need to add some API. I'm going to cut a ticket

Reply:
https://molecula.atlassian.net/browse/CLOUD-1345